src/emailpro/external-contact/add/emailpro-external-contact-add.html

```diff
@@ -1,66 +1,99 @@
-<div data-ng-controller="EmailProAddExternalContactCtrl">
+<div data-ng-controller="EmailProAddExternalContactCtrl as ctrl">
     <div data-wizard
          data-wizard-bread-crumb
          data-wizard-on-cancel="resetAction"
-         data-wizard-on-finish="addContact"
+         data-wizard-on-finish="submitting"
          data-wizard-title="i18n.exchange_tab_EXTERNAL_CONTACTS_configuration_contact_add_title">
 
         <div data-wizard-step
-             data-wizard-step-on-load="init"
-             data-wizard-step-valid="accountIsValid()">
+             data-wizard-step-valid="ctrl.externalContactAddForm.emailAddress.$dirty && ctrl.externalContactAddForm.emailAddress.$valid">
 
-            <p class="mb-3" data-i18n-static="exchange_tab_EXTERNAL_CONTACTS_configuration_contact_add_step1_question"></p>
-            <form class="form-horizontal" name="externalContactAddForm">
+            <div class="loader text-center" data-ng-if="ctrl.isLoading"></div>
+
+            <form class="form-horizontal" name="ctrl.externalContactAddForm" data-ng-if="!ctrl.isLoading">
                 <p>
                     <small class="text-danger">*</small>
                     <small data-i18n-static="required_fields"></small>
                 </p>
-                <div class="form-group" data-ng-class="getPasswordInvalidClass()">
-                    <label for="emailAddress" class="control-label col-md-3 required"
+
+                <div class="form-group" data-ng-class="{'has-error': ctrl.externalContactAddForm.emailAddress.$dirty && ctrl.externalContactAddForm.emailAddress.$invalid}">
+                    <label for="emailAddress"
+                           class="control-label col-md-3 required"
                            data-i18n-static="exchange_tab_EXTERNAL_CONTACTS_configuration_contact_add_step1_email_label"></label>
                     <div class="col-md-9">
-                        <input class="form-control" id="emailAddress" type="text"
-                               data-ng-model="model.newAccount.externalEmailAddress">
-                        <span class="help-block text-danger"
-                              data-ng-if="!isEmailValid() && model.newAccount.externalEmailAddress"
-                              data-i18n-static="exchange_tab_EXTERNAL_CONTACTS_configuration_contact_add_step1_email_invalid"></span>
+                        <input class="form-control"
+                               id="emailAddress"
+                               name="emailAddress"
+                               type="text"
+                               min="0" data-ng-min="0"
```

Comment:
`data-ng-min` is sufficient ;)

Reply:
`data-ng-min` doesn't trigger the brower validation, so the user could use the number input to get below -1. The validation would prevent submitting but with the `min` attribute, the use wouldn't even be able to go below 0.